config/app/api/openapi/registry.yml

```diff
@@ -0,0 +1,117 @@
+openapi: 3.0.0
+info:
+    version: 1.0.0
+    contact:
+        name: Spryker
+        url: 'https://apps.spryker.com/'
+        email: support@spryker.com
+    title: 'Spryker API'
+    license:
+        name: MIT
+paths:
+    '/private/application-config':
+        post: &paths/private/application-config/post
+            summary: 'Saves or updates application configuration between Tenant and App.'
+            parameters:
+                -   name: X-Store-Reference
+                    in: header
+                    required: true
+                    description: 'Reference of the Tenants store.'
+                    schema:
+                        type: string
+            requestBody:
+                description: 'Provides configuration for the application.'
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/ConfigurationApiRequest'
+            responses:
+                200:
+                    description: 'Expected response to a valid request.'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ConfigurationApiResponse'
+                400:
+                    description: 'Bad request.'
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ApiErrorMessages'
+                default:
+                    description: 'Expected response to a bad request.'
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+        put: *paths/private/application-config/post # $ref is not (officially) supported for operation objects
+
+    '/private/application/disconnect':
```

Comment:
I would like to see both endpoints be uniform in their usage of slashes and dashes.

```
    '/private/application-config':
    '/private/application/disconnect':
```
Do we even want to include the `/private` prefix? @stereomon

Reply:
AFAIK that is what we need to have. @matweew this is still true, correct?

Reply:
it's still true, nothing changed

Reply:
> uniform in their usage of slashes and dashes.
unfortunately this isn't possible since module part of the URL (`application-config` and `application` in this case) is generated based on the application module itself; for all current PBCs we have 2 modules - one with business logic and one with configuration logic

Reply:
@bohdan-turchyk-spryker as we discussed, we need to use Glue controllers, where you can defined the URL